bundles/core/src/main/java/com/adobe/cq/wcm/core/components/util/ComponentUtils.java

```diff
@@ -192,4 +194,14 @@ private static String generateId(@NotNull final Resource resource,
     public static String generateId(@NotNull final String prefix, @NotNull final String path) {
         return StringUtils.join(prefix, ID_SEPARATOR, StringUtils.substring(DigestUtils.sha256Hex(path), 0, ID_HASH_LENGTH));
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the resource holding the properties of the featured image of the page.
+     *
+     * @param page the page
+     * @return the featured image resource
+     */
+    public static Resource getFeaturedImage(@NotNull Page page) {
```

Comment:
add @Nullable

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1574/commits/50dfbb9065dd5d10eb4d6cf268d757815b4a6543